.circleci/config.yml

```diff
@@ -270,6 +270,8 @@ jobs:
       - run:
           name: Cleanup build regression folder
           command: rm -r ./build-regression
+      - store_artifacts:
+          path: tmp/screenshots
```

Comment:
Is this configured somewhere? I think the default output directory of playwright trace report is called "test-results"

Reply:
Oooh good call I didn't push my most recent commit to main thanks!!